pset-functions.js

```diff
@@ -174,7 +268,23 @@ Use the following test cases to confirm your program meets the success criteria
 console.log('Problem 4:')
 
 // Add your code below this line
-
+function isPalindrome (word) {
+  if (word === reverseWord(word)) {
+    return true
+  } else {
+    return false
+  }
+}
+
+function reverseWord (fwd) {
+  const str = fwd.split('')
+  const reverseStr = str.reverse()
+  const reversedWord = reverseStr.join('')
+
+  return reversedWord
+}
+
+console.log(isPalindrome('run'))
```

Comment:
@sad3xoxo Nice work, Good job creating the function to check to see if the value is palindrome and good job creating the function `reverWord` to handle returning the reversed string.
Take a look at this solution [here](https://github.com/jsr20190826/hw-02-functions-solution/blob/a4d789161886ff115657133c279062df8a4ca72d/pset-functions.js#L242)